scripts/if_changed.sh

```diff
@@ -45,6 +45,7 @@ elif [[ -z "$TRAVIS_COMMIT_RANGE" ]]; then
 
 else
   case "$PROJECT-$METHOD" in
+
```

Comment:
Should this be removed? Maybe was an accidental addition

Reply:
Done.